.travis.yml

```diff
@@ -63,59 +63,59 @@ matrix:
       env: OS="Ubuntu 12.04" CXX_V=5 CC_V=5 EXPORT=true
       addons:
         apt:
-            packages:
-                - g++-5
-                - gcc-5
-            sources:
-                - ubuntu-toolchain-r-test 
+          packages:
+            - g++-5
+            - gcc-5
+          sources:
+            - ubuntu-toolchain-r-test
 
     - sudo: false
       compiler: g++-4.8
       env: OS="Ubuntu 12.04" CXX_V=4.8 CC_V=4.8 EXPORT=true
-      addons: 
+      addons:
         apt:
-            packages:
-                - g++-4.8
-                - gcc-4.8
-            sources:
-                - ubuntu-toolchain-r-test 
+          packages:
+            - g++-4.8
+            - gcc-4.8
+          sources:
+            - ubuntu-toolchain-r-test
 
     - sudo: false
       compiler: clang-3.6
       env: OS="Ubuntu 12.04" VALGRIND=true CXX_V=3.6 CC_V=3.6 EXPORT=true
       addons:
         apt:
-            packages:
-                - clang-3.6
-                - llvm-3.6-dev
-                - valgrind
-            sources:
-                - llvm-toolchain-precise-3.6
-                - ubuntu-toolchain-r-test
+          packages:
+            - clang-3.6
+            - llvm-3.6-dev
+            - valgrind
+          sources:
+            - llvm-toolchain-precise-3.6
+            - ubuntu-toolchain-r-test
 
     - sudo: false
-      compiler: clang-3.6 
-      env: OS="Ubuntu 12.04" COVERAGE=true CXX_V=3.6 CC_V=3.6 EXPORT=true 
+      compiler: clang-3.6
+      env: OS="Ubuntu 12.04" COVERAGE=true CXX_V=3.6 CC_V=3.6 EXPORT=true
       addons:
         apt:
-            packages:
-                - clang-3.6
-                - llvm-3.6-dev
-                - valgrind
-            sources:
-                - llvm-toolchain-precise-3.6
-                - ubuntu-toolchain-r-test
+          packages:
+            - clang-3.6
+            - llvm-3.6-dev
+            - valgrind
+          sources:
+            - llvm-toolchain-precise-3.6
+            - ubuntu-toolchain-r-test
 
     - dist: trusty
-      compiler: clang-3.5
+      compiler: clang-3.6
       env: OS="Ubuntu 14.04"
 
     - dist: trusty
-      compiler: g++-4.8
+      compiler: g++-5
       env: OS="Ubuntu 14.04"
 
     - os: osx
-      compiler: g++-4.8
+      compiler: g++-5
```

Comment:
Cool, with g++-5 [the osx build succeed!](https://travis-ci.org/measurement-kit/measurement-kit/jobs/118298451)